pkg/cli/run_test.go

```diff
@@ -12,9 +13,9 @@ import (
 
 func ArgsToApp(t *testing.T, args ...string) *apiv1.App {
 	t.Helper()
-
+	client := &client.CommandContext{}
 	buf := &bytes.Buffer{}
-	cmd := NewRun(buf)
+	cmd := NewRun(buf, *client)
```

Comment:
Any reason to create `client` as a pointer here? We just dereference it when we pass it to `NewRun()`

Reply:
no reason I thought it was used differently in the NewRun but changed from pointer